<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-4G1ICOQM" failures="0" tests="6" name="com.RestaurentTests.HomeTest" time="15.842" errors="1" timestamp="2022-07-05T16:48:31 IST" skipped="4">
  <testcase classname="com.RestaurentTests.HomeTest" name="validateExcel" time="7.800">
    <error message="C:\Users\chinm\git\Selenium\RestaurentProject\src\main\resources (Access is denied)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\Users\chinm\git\Selenium\RestaurentProject\src\main\resources (Access is denied)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at com.DataHandlers.ExcelUtility.setExcelFileSheet(ExcelUtility.java:32)
at com.RestaurentTests.HomeTest.validateExcel(HomeTest.java:95)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- validateExcel -->
  <system-out/>
  <testcase classname="com.RestaurentTests.HomeTest" name="peopleSubWaiters" time="8.042"/>
  <system-out/>
  <testcase classname="com.RestaurentTests.HomeTest" name="validateMainNavigationLinks">
    <skipped/>
  </testcase> <!-- validateMainNavigationLinks -->
  <system-out/>
  <testcase classname="com.RestaurentTests.HomeTest" name="validateUserDetails">
    <skipped/>
  </testcase> <!-- validateUserDetails -->
  <system-out/>
  <testcase classname="com.RestaurentTests.HomeTest" name="peopleSuppliers">
    <skipped/>
  </testcase> <!-- peopleSuppliers -->
  <system-out/>
  <testcase classname="com.RestaurentTests.HomeTest" name="peopleCustomers">
    <skipped/>
  </testcase> <!-- peopleCustomers -->
  <system-out/>
</testsuite> <!-- com.RestaurentTests.HomeTest -->
